pkg/goDB/conditions/node/filter_direction.go

```diff
@@ -0,0 +1,160 @@
+package node
+
+import (
+	"errors"
+	"fmt"
+	"github.com/els0r/goProbe/pkg/types"
+	"github.com/els0r/goProbe/pkg/types/hashmap"
+)
+
+// FilterTypeDirection denotes filters wrt. directionality of traffic
+type FilterTypeDirection string
+
+const (
+	filterTypeDirectionIn         FilterTypeDirection = "in"
```

Comment:
How about exporting the unsugared ones, then they can be used e.g. in the E2E tests (and / or elsewhere) without having to type out the strings again?

Reply:
Absolutely agree. I am currently using the `FilterTypeDirection...` constants to include the newly added `dir` keyword in the autocompletion. I would suggest to move it into `types` for consistency with other column/type names?

Reply:
By all means! :+1: